File: src/Model/BearTamer.java
package Model;

public class BearTamer extends Character {

    public BearTamer() {
        super("BearTamer",
                "Each morning, if the Bear Tamer is sitting to the right or left of a werewolf, the bear growls (indicated by the Game Master). If the Bear Tamer is infected by the Werewolf Elder, the bear will growl until the end of the game.",
                false,
                PowerEnum.Constant,
                "../../resources/images/characters/BearTamer.jpg");
    }
}

File: app/src/main/java/Model/BearTamer.kt
package Model

class BearTamer(order: Int)
    : Character("The BearTamer can signal if one of his neighbours is a werewolf. If the game master makes a sound like a bear growl, that means one of the alive neighbours of the BearTamer is a werewolf. The Bear also growls if the BearTamer is infected."
    , false
    , PowerState.PERMANENT
    , false
    , order
    , 1) {

    override fun clone() = BearTamer(this.order)
}